test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ComplexTypeInheritance/ComplexTypeInheritanceTests.cs

```diff
@@ -112,6 +112,8 @@ protected override void UpdateConfiguration(WebRouteConfiguration configuration)
                 response.StatusCode,
                 requestUri,
                 contentOfString));
+
+            Assert.Equal("4.0", response.Headers.GetValues("OData-Version").Single());
```

Comment:
This is a POST request with response code 201, so it is not related to the case of 204 here, right?

Reply:
Thanks. That's not added from my change. That's added in the OData Formatter, for example in Core version, it's added from "https://github.com/OData/WebApi/blob/master/src/Microsoft.AspNetCore.OData/Formatter/ODataOutputFormatter.cs#L183"

I added this code to verify the header.